### PR TITLE
feat: support arm64

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @hyy0322 @liwandaniel 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: build/web/Dockerfile.arm64.local [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM arm64v8/nginx:1.14-alpine
+LABEL maintainer="chende@caicloud.io"
```
Comment (supereagle) [inline build/web/Dockerfile.arm64.local]: Update maintainer

File: build/workflow/controller/Dockerfile.arm64 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM arm64v8/alpine:3.8
+
+LABEL maintainer="chende@caicloud.io"
```
Comment (supereagle) [inline build/workflow/controller/Dockerfile.arm64]: Same here

File: build/cicd/cd/Dockerfile.arm64 [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM arm64v8/alpine:3.8
+
+LABEL maintainer="chende@caicloud.io"
```
Comment (supereagle) [inline build/cicd/cd/Dockerfile.arm64]: Update maintainer

File: build/web/Dockerfile.arm64.local (hunk 1 above)
Comment (zhujian7) [reply build/web/Dockerfile.arm64.local]: done

File: build/cicd/sonarqube/Dockerfile.arm64 [hunk 4]
```diff
@@ -0,0 +1,50 @@
+FROM arm64v8/openjdk:8-alpine3.8
+
+LABEL maintainer="Ryan Mitchell <mitch@ryansmitchell.com>"
```
Comment (liwandaniel) [inline build/cicd/sonarqube/Dockerfile.arm64]: 这里可以去掉

Comment (zhujian7) [reply build/cicd/sonarqube/Dockerfile.arm64]: PTAL

**Pull request merged**